project/Templates/procedure-software-template.rst

```diff
@@ -1,6 +1,6 @@
-##################
-Title of procedure
-##################
+###########################
```

Comment:
This template doesn't have the form of the others with the labels, explanations and warnings. What kind of operational procedures will be included in the "procedure-software"? I see most or all of them under the "Procedure-template" below.

Reply:
👍 Removed. It was a leftover from previous templates.